components/Callout/index.jsx

```diff
@@ -4,7 +4,6 @@ const PropTypes = require('prop-types');
 const Callout = props => {
   let { children } = props;
   const { attributes, theme, title, icon } = props;
-  if (!(title || children)) return '';
```

Comment:
In a previous PR, I pushed the test coverage below the threshold. In trying to write some tests to increase coverage, I found this line. `children` is required in `propTypes` so this _should_ never be true.

Reply:
lol "Remove impossible line"